tests/Fixtures/TestBundle/Entity/Dummy.php

```diff
@@ -130,6 +140,13 @@ class Dummy
      */
     public $nameConverted;
 
+    /**
+     * @var bool
+     *
+     * @ORM\Column(type="boolean", nullable=true, name="existsd")
```

Comment:
Hmm, something that would prevent from doing a NumericFilter on boolean for example?

I think that would be great indeed, with nice error messages to guide the developer.